lib/config/executor/network_interface.go

```diff
@@ -60,17 +60,26 @@ type ContainerNetwork struct {
 
 	// The network scope the IP belongs to.
 	// The IP address is the default gateway
-	Gateway net.IPNet `vic:"0.1" scope:"read-write" key:"gateway"`
+	Gateway net.IPNet `vic:"0.1" scope:"read-only" key:"gateway2"`
 
 	// Should this gateway be the default route for containers on the network
 	Default bool `vic:"0.1" scope:"read-only" key:"default"`
 
 	// The set of nameservers associated with this network - may be empty
-	Nameservers []net.IP `vic:"0.1" scope:"read-write" key:"dns"`
+	Nameservers []net.IP `vic:"0.1" scope:"read-only" key:"dns2"`
 
 	// The IP ranges for this network
 	Pools []ip.Range `vic:"0.1" scope:"read-only" key:"pools"`
 
 	// set of network wide links and aliases for this container on this network
 	Aliases []string `vic:"0.1" scope:"hidden" key:"aliases"`
+
+	Assigned struct {
```

Comment:
then we'll need a definition for the structure. otherwise, we have no idea for which attributes should be considered in upgrade